src/main/java/seedu/duke/PersonList.java

```diff
@@ -0,0 +1,42 @@
+package seedu.duke;
+
+import java.util.ArrayList;
+
+public class PersonList {
+    private static ArrayList<Person> list;
+
+    public PersonList(){
+        list = new ArrayList<Person>();
+    }
+
```

Comment:
I think we should focus on a single profile first and get our app working on that front instead of branching out and extending it to support multiple profiles. That can be done further down the road.

Reply:
Yeah I do agree, this part can get along with other parts anyway. I'm just trying to build a skeleton in case of complicated changes in the future. Right now we should focus more on the basic needs.